components/Tabs/tests/snapshot/Theming/MDCTabBar_MaterialThemingSnapshotTests.m

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialSnapshot.h"
+
+#import "MaterialContainerScheme.h"
+#import "MaterialTabs+Theming.h"
+#import "MaterialTabs.h"
+
+static NSString *const kItemTitle1 = @"Item 1";
+static NSString *const kItemTitle2 = @"Item 2";
+static NSString *const kItemTitle3 = @"Item 3";
+static NSString *const kItemBadgeValue = @"Badge";
+
+/** Snapshot tests for MDCTabBar rendering .*/
+@interface MDCTabBar_MaterialThemingSnapshotTests : MDCSnapshotTestCase
+
+/** The view being tested. */
+@property(nonatomic, strong) MDCTabBar *tabBar;
+@property(nonatomic, strong) UITabBarItem *item1;
+@property(nonatomic, strong) UITabBarItem *item2;
+@property(nonatomic, strong) UITabBarItem *item3;
+@property(nonatomic, strong) MDCContainerScheme *containerScheme;
+@end
+
+@implementation MDCTabBar_MaterialThemingSnapshotTests
```

Comment:
Please add tests that include images and at least one with a badge.

Reply:
work in progress... (to be updated)

Reply:
Addressed in 1d0441f7dc767c1a545ffecd3df1ff42f7ea7426.